test/voice-broadcast/components/molecules/__snapshots__/VoiceBroadcastPlaybackBody-test.tsx.snap

```diff
@@ -326,7 +485,7 @@ exports[`VoiceBroadcastPlaybackBody when rendering a stopped broadcast and the l
       <span
         class="mx_Clock"
       >
-        00:42
+        23:42
```

Comment:
This implies an unexpected change?

Reply:
No, this is expected. I've added another test to explicitly look for the time update.

https://github.com/matrix-org/matrix-react-sdk/pull/9592/files#diff-6e97d35a4cfa5d17d4be2ea0e889b83b6081db3fbd23c7cc07333324837b14e8R135

comes from here

https://github.com/matrix-org/matrix-react-sdk/pull/9592/files#diff-6e97d35a4cfa5d17d4be2ea0e889b83b6081db3fbd23c7cc07333324837b14e8L107